docs/topics/forms/modelforms.txt

```diff
@@ -972,6 +967,27 @@ so long as the total number of forms does not exceed ``max_num``::
 A ``max_num`` value of ``None`` (the default) puts a high limit on the number
 of forms displayed (1000). In practice this is equivalent to no limit.
 
+Also, ``extra=0`` doesn't prevent creation of new model instances as you can
+:ref:`add additional forms with JavaScript <understanding-the-managementform>`
+or just send additional POST data.
+
+Edit-only mode for the model formsets
+---------------------------------------
+
+.. versionadded:: 4.0
+
+.. attribute:: BaseModelFormSet.edit_only
+
+You can use the Model Formset in ``edit_only`` mode, which will block creation of
+any new instances of a model.::
+
+    >>> AuthorFormSet = modelformset_factory(Author, fields=('name',), extra=2)
```

Comment:
I guess my question is should I be able to pass in `edit_only=True` here?

Reply:
@smithdc1, maybe, it's better place for this argument. I'm not sure about the design of framework, but `edit_only` is close in meaning to other arguments of  `modelformset_factory`.

Reply:
@smithdc1, what do you think? Should I place `edit_only` to `modelformset_factory`?

Reply:
It is what I'd expect but I feel I am out my my depth to provide specific guidance. Maybe I could ask @jacobtylerwalls To opine on this. 

I also noticed there's also a merge conflict with the release notes (which is a bit odd) so could you rebase? 

Thanks for your continued efforts with this patch. 🙏

Reply:
Yes, seems wise to add it. Eventually someone will open a ticket saying it doesn't work. :P

> it's better place for this argument

David isn't asking you to *move* anything, just to add the keywords to the pipeline starting from `modelformset_factory()` leading down to where you already have these keywords.

Reply:
Jacob, thank you for taking the time to look at this. 👍

Reply:
Sorry, @smithdc1. Maybe I don't understand. You suggested me to add `edit_only` kwarg to `modelformset_factory` and not remove it from `BaseModelFormset` class. The result will be this method of creating "edit-only modelformset":
`AuthorFormSet = modelformset_factory(Author, fields=('name',), extra=2, edit_only=True)`
`formset = AuthorFormSet(edit_only=True)`
I don't understand why dublicate kwargs. Most likely I misunderstood you. Can you explain in more detail?

Reply:
I think the example would be something like this, with the kwarg only passed to the factory. Code wise, you'll need it in both places so it can be passed through. 

> `AuthorFormSet = modelformset_factory(Author, fields=('name',), extra=2, edit_only=True)`

> `formset = AuthorFormSet()`

Hope this helps.